README.md

```diff
@@ -35,7 +35,7 @@ npm install electron --save-dev --save-exact
 
 The `--save-exact` flag is recommended as Electron does not follow semantic
 versioning. For info on how to manage Electron versions in your apps, see
-[Electron versioning](https://electronjs.org/docs/tutorial/versioning).
+[Electron versioning](docs/tutorial/versioning.md).
```

Comment:
Asking because I'm not sure, would we want the link to start with `docs` or with `/docs`?

I get & agree that we're trying to not hardcode the domain, but it seems like we'd want to make the path absolute?

Reply:
This page is not used on the website. I believe GH has put work into ensuring relative markdown links work everywhere.

Reply:
> would we want the link to start with docs or with /docs

@ckerr either of these formats will work, both on GitHub and the website.